src/main/java/bio/terra/app/configuration/DuosConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.app.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "duos")
```

Comment:
I think you can replace these with a single annotation
```suggestion
@ConfigurationProperties(prefix = "duos")
```

Reply:
I do think we need `@Configuration`, the app fails to start up without it:

```
***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of constructor in bio.terra.service.duos.DuosClient required a bean of type 'bio.terra.app.configuration.DuosConfiguration' that could not be found.


Action:

Consider defining a bean of type 'bio.terra.app.configuration.DuosConfiguration' in your configuration.
```

But I do think that `@EnableConfigurationProperties` is redundant, I was able to start up the app and see the `duos.basePath` property pulled in.

Reply:
Interesting, in catalog we don't use it. Maybe that's because our spring app uses `@ConfigurationPropertiesScan`, which scans all classes in a package hierarchy for `@ConfigurationProperties` annotated classes.
https://github.com/DataBiosphere/terra-data-catalog/blob/main/service/src/main/java/bio/terra/catalog/App.java#L28